runtime/autoload/health.vim

```diff
@@ -42,24 +42,26 @@ function! health#check(plugin_names) abort
   else
     redraw|echo 'Running healthchecks...'
     for c in healthchecks
-      let output = ''
-      call append('$', split(printf("\n%s\n%s", c, repeat('=',72)), "\n"))
+      let s:output = []
+      let header = [c[0] . ': ' . c[1], repeat('=', 72)]
       try
-        let output = "\n\n".execute('call '.c.'()')
+        if c[1] == ''
+          throw 'healthcheck not found'
```

Comment:
`c[1]` is already used above, why do this check here? also weird to throw in this try-catch